packages/backend/src/.editorconfig

```diff
@@ -0,0 +1,256 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = false
```

Comment:
Usually it is a good practice to have new line in the EOF

https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file